mobile/src/navigations/root/root.tsx

```diff
@@ -31,18 +31,10 @@ const Root: React.FC = () => {
     const { isProfileComplete, role } = userData ?? {};
     const dispatch = useAppDispatch();
 
-    const { dataStatus } = useAppSelector(({ auth }) => auth);
-
-    const isPendingAuth = dataStatus === DataStatus.PENDING;
-
```

Comment:
it should stay here, user shouldn't see sign in screen each time app loads
here we should return Loader (white screen with spinner) 
in auth form we'll return overlay